docs/papers-publication/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: papers-publications
+show_sidebar: false
+permalink: /papers-publications/
+
+# Banner Content
+# =================================
+title: Papers and Publications
+subtitle: We have been involved in these and related research areas for several years
+hero_image: /assets/img/home-callouts/papers_publications.jpg
+
+# Papers and Publications list
+# =================================
+publications:
+  - title: Machine-based detection and classification for bone marrow aspirate differential counts - initial development focusing on nonneoplastic cells.
```

Comment:
I find the spacing of this page very large (I expect a list of publications to be denser), but it is just an opinion.

Reply:
fixed in 505f8c1041fbedb1fa712528f58733dc40d3203c